### DISCUSSION TITLE
Get "Common Name" of the certificate installed by match

### DISCUSSION THREAD

**Discussion by serbixote** [fastlane/fastlane#18254] [Q&A]
Hey guys!

I'm using `match/sync_code_signing` along with `gym/build_app`. So far I'm able to set the provisioning profile by getting it from `ENV["sigh_<app identifier>_appstore_profile-name"]`, but I'm unable to get the certificate name to use it as `codesigning_identity` parameter for `gym/build_app`.

I would say that a minimum desire would be to get it in a similar way as the provisioning profile name, and ideally  `match/sync_code_signing` should set those values automatically.

I'm really new to Fastlane so I may be missing something.

Thanks! 🙏🏼

**Comment by magnoahlia** [comment]
I'm interested in this, too! I am setting up macOS app notarization and would like to use our existing match repo to store the certificates (since we already do this for iOS signing). However, I don't know how to get the certificate name to pass to codesign after it's been installed by match.
